<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDlg</class>
 <widget class="QDialog" name="SearchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="sLblLlcx">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#005500;&quot;&gt;浏览查询&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblRstTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#0055ff;&quot;&gt;最近的记录：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sBtnReturn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回首页</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblSearch">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;查找日志：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblSearchWay">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>查询方式：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sCboxSearchWay">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>按日期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按地点</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按标题</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="sLblSearchDate">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>日期：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="sDetSearchDate">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sLblSearchPlace">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>地点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sLetSearchPlace">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sLblSearchTitle">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>标题：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sLetSearchTitle">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="sTbtnSearch">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblResult1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblResult2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblResult3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>141</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblResult4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblResult5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblResult6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="sTbtnRstInxLast">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="sTbtnRstInxNext">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="sLblRstIndex">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sBtnReturn</sender>
   <signal>clicked()</signal>
   <receiver>SearchDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
